RELEASE_NOTE.md

```diff
@@ -0,0 +1,76 @@
+# Release v0.13.0
+
+
+## New Features
+
+* PAI protocol:
+  - Introduce PAI protocol and job submission v2 ([#2260](https://github.com/microsoft/pai/pull/2260))
+  - Add new job submission v2 plugin ([#2461](https://github.com/microsoft/pai/pull/2461))
+
+* Web portal:
+  - Add new home page ([#2544](https://github.com/microsoft/pai/pull/2544), [#2614](https://github.com/microsoft/pai/pull/2614))
+  - Add new user management page ([#2726](https://github.com/microsoft/pai/pull/2726), [#2796](https://github.com/microsoft/pai/pull/2796))
```

Comment:
User Management UX refactoring with new layout and themes.

Reply:
updated